pkg/util/sys/device.go

```diff
@@ -91,6 +102,59 @@ func ListDevices(executor exec.Executor) ([]string, error) {
 	return strings.Split(devices, "\n"), nil
 }
 
+func GetCephVolumeInventory(executor exec.Executor) (*[]CephVolumeDeviceData, error) {
+	cmd := "ceph-volume inventory --format plain"
+	inventory, err := executor.ExecuteCommandWithOutput(false, cmd, "ceph-volume", "inventory", "--format", "json")
```

Comment:
no need for `cmd`. The 2nd arg can be empty. The intention of that argument in the past was for logging, but I'm not sure we even log it anymore.

Reply:
Removed. thx!